test/UuidGenerator/UuidTest.php

```diff
@@ -39,5 +39,9 @@ public function testGenerateV5()
         $generator2 = new UuidGenerator();
         $uuid5_2 = $generator2->generateV5($object_name, $namespace);
         $this->assertEquals($test_uuid->toString(), $uuid5_2, "Did not build the correct UUID v5");
+
+        $generator_3 = new UuidGenerator($namespace);
```

Comment:
Ok that is if you pass a UUID for the namespace (which you are generating on line32) as the $namespace parameter. So something like
```
$uuid5_3 = $generator2->generateV5($object_name, $namespace_uuid);
```